ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -1067,7 +1276,8 @@ private void verifyOperationsAfterDelete(String blobId, JSONObject expectedHeade
 
     GetOption[] options = {GetOption.Include_Deleted_Blobs, GetOption.Include_All};
     for (GetOption option : options) {
-      getBlobAndVerify(blobId, null, option, expectedHeaders, expectedContent);
+      getBlobAndVerify(blobId, null, option, expectedHeaders, expectedContent, Account.UNKNOWN_ACCOUNT,
```

Comment:
It can, and these cases should be covered in `injectionAccountAndContainerForGetHeadDeleteTest`.